drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-showcase/pom.xml

```diff
@@ -0,0 +1,1153 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2012 Red Hat, Inc. and/or its affiliates.
```

Comment:
Dates need an update here too.

Reply:
Fixed in
[DROOLS-2786] Investigate and create a scenario simulation showcase - Fixing Copyright year